contracts/programs/ocr2/src/lib.rs

```diff
@@ -656,15 +656,15 @@ fn transmit_impl<'info>(ctx: Context<Transmit<'info>>, data: &[u8]) -> ProgramRe
     require!(raw_signatures.len() % SIGNATURE_LEN == 0, InvalidInput);
     let signature_count = raw_signatures.len() / SIGNATURE_LEN;
     require!(
-        signature_count == 3 * usize::from(config.f) + 1,
+        signature_count == usize::from(config.f) + 1,
```

Comment:
signature number should be f + 1

Reply:
[This line in Gauntlet ocr2 transmit cmd](https://github.com/smartcontractkit/chainlink-solana/blob/459f20cf6bb515b4c1893fffe294aed7c271fe6b/gauntlet/packages/gauntlet-solana-contracts/src/commands/contracts/ocr2/transmit.dev.ts#L61) should also be changed.

cc @RodrigoAD